server/api/inspect-url.post.test.ts

```diff
@@ -0,0 +1,78 @@
+import { createHash } from 'crypto'
+import { describe, it, expect, vi, beforeEach } from 'vitest'
+
+// Mock the server modules
+vi.mock('~server/db', () => ({
+  saveRun: vi.fn(),
+}))
+
+vi.mock('#app', () => ({
+  defineEventHandler: (handler: (event: unknown) => unknown) => handler,
+  readBody: vi.fn(),
+  createError: vi.fn(error => new Error(error.message)),
+  $fetch: {
+    raw: vi.fn(),
+  },
+}))
```

Comment:
this can't be the idiomatic way to test a nuxt api endpoint? try again, or just forget about test coverage